examples/send-plugin-message/pluginMessageApp.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Html Messages</title>
+</head>
+
+<body onload="main()">
+</body>
+
+<script>
+
+  var bsMessage = new BSMessagePort();
+
+  function main() {
+    console.log("=== Starting javascript...");
+
+    bsMessage.onbsmessage = function (msg) {
+      if (msg.data["serialNumber"] !== undefined && msg.data["filename"] !== undefined) {
+        console.log("=== Received plugin message from: " + msg.data["serialNumber"]);
+
+        // Add fields from plugin message
+        var serialNumber = msg.data["serialNumber"];
+        var filename = msg.data["filename"];
+
+        // send info to your backend (example)
+        fetch('https://jsonplaceholder.typicode.com/posts', {
+          method: 'POST',
+          body: JSON.stringify({
+            serialNumber,
+            filename
+          })
+        })
```

Comment:
I know we talked about this for another example, though for this example, sending to jsonplaceholder.com may not make sense. Unless there's other possible interesting ideas, I suggest a simple log may get the point across well.